sros2/test/sros2/commands/security/verbs/test_generate_policy.py

```diff
@@ -12,133 +13,155 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import itertools
 import os
+import sys
 import tempfile
 
+from launch_ros.actions import Node
```

Comment:
this should be added as a dependency in the package.xml

Reply:
`ros_testing` will bring the entire `launch(_ros)?` and `launch_testing(_ros)?` ecosystem (that's the point of it), but we can be explicit if you'd like.

Reply:
In general we aim for "declare everything you use directly" to avoid the effects of dependencies disappearing under the hood and breaking us. But if the raison-d'etre of the `ros_testing` package is to give a single entrypoint for all things testing and always provide all these dependencies then I guess that's fine.

Can you please add a comment to that effect in the package.xml ? this will avoid future us from pruning the dependency